tests/e2e/specs/menu-top/MenuTop.spec.js

```diff
@@ -67,5 +67,27 @@ describe('components/menu-top/MenuTop', () => {
       cy.get('[data-test="grapher--grid-horizontal"]')
         .should('not.exist');
     });
+
+    it('can toggle yardlines', () => {
+      cy.get('[data-test="grapher--yard-line"]')
+        .should('be.visible');
```

Comment:
Feel free to make this and the others below one liners since it isn't too big of a line